### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by shaunbent**
@simonsinclair spotted that documentation for the 'Output Configuration' variables was incorrect/missing.

### CODE REVIEW

File: lib/_settings.scss [hunk 1]
```diff
@@ -40,24 +40,25 @@ $gel-type-no-touch-group: 'group-d';
...
 //
-// @link http://bit.ly/1Z6hPfd
-//
-// @type String
+// @type Bool
 //
 $gel-type-enable--markup-output: false !default;
 
-// Which typography group contains our non-touch typography styles
+// Barlesque will set the correct font-family on the <body> tag but
```
Comment (azizur) [inline lib/_settings.scss]: Given this is an open source project, we should make it clear that `Barlesque` is the BBC's header and footer navigation. If someone is using this library on a non BBC project what they need to do. This should also be documented in the `README.md` docs.

Comment (shaunbent) [inline lib/_settings.scss]: I've updated these.

**Comment by azizur** [comment]
Looks good to me. :+1: :+1:

**Comment by simonsinclair** [comment]
Looks good, @shaunbent.

**Comment by shaunbent** [comment]
Cheers guys! :smile:

**Pull request merged**